From 26fa2a8ecaedcb47e02ae26bf8ab4f02b6861099 Mon Sep 17 00:00:00 2001
From: Hans Petter Selasky <hps@selasky.org>
Date: Sat, 4 Apr 2020 03:29:48 +0200
Subject: [PATCH 1507/1936] The noteFreq value in ModFilter doesn't have to be
 smoothed, because all the places the noteFreq is read, it is already assigned
 to a smoothed variable.

Signed-off-by: Hans Petter Selasky <hps@selasky.org>
---
 src/Synth/ModFilter.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Synth/ModFilter.h b/src/Synth/ModFilter.h
index 1dca0c63..8370c2ff 100644
--- a/src/Synth/ModFilter.h
+++ b/src/Synth/ModFilter.h
@@ -55,7 +55,7 @@ class ModFilter
 
         smooth_float baseQ;    //filter sharpness
         smooth_float baseFreq; //base filter frequency
-        smooth_float noteFreq; //frequency note was initialized to
+        float noteFreq; //frequency note was initialized to
         smooth_float tracking; //shift due to note frequency
         smooth_float sense;    //shift due to note velocity
 
-- 
2.47.0

